docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,66 @@
+# Capture
+
+
+The Dali::Capture is a class to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.
+
+
+## How to capture your scene
+
+Applications should follow the example below to use `Dali::Capture`.
```

Comment:
Please rephrase for consistency and add a colon at the end.

The following example shows how to use `Dali::Capture`:

Reply:
Thank you it is better!